src/routes/event-match.tsx

```diff
@@ -98,14 +98,18 @@ const EventMatch = ({ eventKey, matchKey }: Props) => {
         m.group +
         (m.num ? ' Match ' + m.num : '') +
         ' - ' +
-        (event ? event.name : eventKey)
+        (isData(event) ? event.name : eventKey)
```

Comment:
@j10k11 I think we can leave out the changes to this file so Victor can handle them in his PR

Reply:
Just a reminder about this feedback. Victor is taking care of all the changes to the match page in his PR, so you can undo the changes here.